### PR TITLE
[HOTFIX] Set default header for load data in legacy enviroment

### PR DESCRIPTION
**Pull request by QiangCai**
### Why is this PR needed?
 The old code has been commented, this PR will revert the code.
 
 ### What changes were proposed in this PR?
Set default header for load data in legacy enviroment
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

File: integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala [hunk 1]
```diff
@@ -212,7 +212,7 @@ object CommonLoadUtils {
       options: Map[String, String]): util.Map[String, String] = {
     val tableProperties = table.getTableInfo.getFactTable.getTableProperties
     val optionsFinal = LoadOption.fillOptionWithDefaultValue(options.asJava)
-    //    EnvHelper.setDefaultHeader(sparkSession, optionsFinal)
+    EnvHelper.setDefaultHeader(SparkSQLUtil.getSparkSession, optionsFinal)
```
Comment (ajantha-bhat) [inline integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala:215]: please add a comment here that it is for legacy store. 

All data load in new store success without this change. so I commented this. If there was a comment why this is required. I would have kept it.

Comment (QiangCai) [reply integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala:216]: done

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1274/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2993/

**Comment by ajantha-bhat** [comment]
LGTM

**Pull request closed without merging**